tests/ssg_test_suite/xml_operations.py

```diff
@@ -122,27 +122,32 @@ def remove_machine_remediation_condition(root):
 
 
 def remove_bash_machine_remediation_condition(root):
-    query = BENCHMARK_QUERY + '//xccdf-1.2:fix[@system="urn:xccdf:fix:script:sh"]'
-    fix_elements = root.findall(query, PREFIX_TO_NS)
+    system = "urn:xccdf:fix:script:sh"
     considered_machine_platform_checks = [
         r"\[\s+!\s+-f\s+/\.dockerenv\s+\]\s+&&\s+\[\s+!\s+-f\s+/run/\.containerenv\s+\]",
     ]
-    for el in fix_elements:
-        if not el.text:
-            continue
-        for check in considered_machine_platform_checks:
-            el.text = re.sub(check, "true", el.text)
+    repl = "true"
```

Comment:
It is not consistent with the line [138](https://github.com/ComplianceAsCode/content/pull/9347/files#diff-414bcf1a7602826f6b3856ff50d4d9809457bd22a7a332e6362df404d749cca1R138).

Reply:
Actually, this is fine for the bash equivalent function. Never mind!